packages/angular-output-target/src/generate-angular-component.ts

```diff
@@ -149,8 +149,14 @@ export const createComponentTypeDefinition = (
   });
   const eventTypes = publicEvents.map((event) => {
     const comment = createDocComment(event.docs);
+    let eventName = event.name;
+    if (event.name.indexOf('-') !== -1) {
```

Comment:
No action required, this could be shortened to
```suggestion
    if (event.name.includes('-')) {
```